### PR TITLE
[DESK][THEMEUI] Move dialog resources from desk.cpl to themeui.dll

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
Move dialog resources to improve compatibility.
JIRA issue: [CORE-10196](https://jira.reactos.org/browse/CORE-10196)
after

### CODE REVIEW

**Comment by HBelusca** [comment]
Can we somehow leave default fallback resources in desk.cpl so that if themeui fails to be loaded (because say you test desk.cpl on an older window version that doesn't have this dll, as very often people do), we can indeed use these fallback resources instead? This would also allow removing these assertions you added.

**Comment by katahiromz** [comment]
I introduced a fallback mechanism with using `DoLoadDialog` function. This function will return `NULL` if the dialog resource is not found in the instance.

**Comment by yagoulas** [comment]
I don't think this solves any problem. To the contrary it will make it more difficult to test desk.cpl on windows. In windows the functionality of desk.cpl is a split in multiple libraries solely for historic reasons, as more tabs were added, they were implemented in different dlls but we won't gain anything copying it in such details. 
Imo stuff like applications or control panel controls never create compatibility problems with windows so we have the freedom to implement them in any way we like (and keeping it as one module makes perfect sense).

**Comment by katahiromz** [comment]
Cc @gonzoMD

File: dll/win32/themeui/lang/bg-BG.rc [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    LTEXT "Theme:", IDC_STATIC, 5, 42, 55, 10
+    COMBOBOX IDC_THEMES_COMBOBOX, 5, 52, 160, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    PUSHBUTTON "Save &As...", IDC_THEMES_SAVE_AS, 170, 52, 70, 14
+    PUSHBUTTON "&Delete", IDC_THEMES_DELETE, 170, 70, 70, 14
+    CONTROL "", IDC_THEMES_PREVIEW, "STATIC", SS_BITMAP, 5, 90, 235, 115, WS_EX_CLIENTEDGE
+END
+
+IDD_SCREENSAVER DIALOGEX 0, 0, 246, 202
+STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
+CAPTION "�E�{���p�~�u�~ �����u�t���p�x�y���u�|"
```
Comment (HBelusca) [inline dll/win32/themeui/lang/bg-BG.rc]: Encoding problem.

File: dll/win32/themeui/lang/et-EE.rc [hunk 2]
```diff
@@ -0,0 +1,203 @@
+/*
+ * PROJECT:     ReactOS Display Control Panel
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Estonian Language resource file
+ * TRANSLATOR:  Joann Mﾃｵndresku (joannmondresku at gmail dot com)
```
Comment (HBelusca) [inline dll/win32/themeui/lang/et-EE.rc]: Encoding problem.

**Comment by gonzoMD** [comment]
I think it is OK to keep it as is, if there are no problems. I just spotted the difference of the resources and then created the bug report when I worked on that applet.

**Comment by katahiromz** [comment]
Thanks and sorry.

**Pull request closed without merging**